testing/btest/language/event-ts-scheduled.zeek

```diff
@@ -0,0 +1,18 @@
+# @TEST-EXEC: zeek -b -r $TRACES/ticks-dns-1hr.pcap %INPUT > out
+# @TEST-EXEC: btest-diff out
+
+global runs = 0;
+
+event test()
+	{
+	print fmt("[%D] Test was scheduled at %D", network_time(), current_event_time());
```

Comment:
Because this is a scheduled event, I *think* this should say "scheduled for" instead of "scheduled at" ? It could be interesting to add the scheduled_at value by passing `network_time()` as an event parameter to `test()` and outputting that, too.  Passing the interval as event parameter and including it in the output could be helpful when trying to make sense of the baseline, as well :-)

Reply:
I like that idea. Makes it indeed easier to read the baseline.